Local testing set up, removed all lint warnings.

zerogerc opened the pull request:
You could run backend + frontend locally now.

zerogerc committed:
Local testing set up, lint warnings

danilkolikov commented on src/api/routes/linksRoutes.js:
```diff
@@ -1,14 +1,15 @@
 
 import linksController from '../controllers/linksController'
-import express from 'express'
+import type {$Application, } from 'express'
```
Split on few lines, or remove comma

danilkolikov commented on src/api/controllers/linksController.js:
```diff
@@ -1,25 +1,45 @@
+  },
+  (err: string, task: string) => {
+    if (err) { res.send(err) }
+    res.json({ message: 'Link successfully deleted', })
```
I think we should decide on the structure of responses from api

danilkolikov commented on src/api/controllers/linksController.js:
```diff
@@ -1,25 +1,45 @@
 
-exports.listAllLinks = (req, res) => {
-  Link.find({}, (err, task) => {
+function appendHeader(res: $Response) {
```
Looks like a crutch. Probably it's possible to set default headers in `express`

Commit:
Absolute paths + __tests__

- Support for absolute paths was added
- Tests were moved to __tests__ subdirectory

zerogerc committed:
default headers

danilkolikov reviewed (approved):
lgtm
